apps/glusterfs/operations_manage.go

```diff
@@ -97,6 +98,7 @@ func AsyncHttpOperation(app *App,
 		// decrement the op counter once the operation is done
 		// either success or failure
 		defer app.opcounter.Dec()
+		time.Sleep(10 * time.Second)
```

Comment:
do we still need this Sleep?  I think it may add an extra delay.  if its required can it be reduced to a smaller value?

Reply:
I think that's just there to help prove the point and would not be part of a long term change.

Reply:
may be I am missing something,  can you please explain  how this delay helps here ?

Reply:
@raghavendra-talur  correct me if I am wrong, but I think the point is that it makes the problem worse by prolonging the TCP connection in order to churn through the ports more quickly. Like I mentioned after @Madhu-1 commented I doubt this will be part of any PR that actually gets merged.

Reply:
@Madhu-1 @humblec 

@phlogistonjohn is right, the PR in current state is only to demonstrate the bug. By increasing the time server spends on servicing the request I was able to clearly see the increase in port consumption by the client. 

The fix would be to embed the http client object in the heketi client object. When done properly it will have the TLS cert config also considered.  The question is, do we all agree with the approach?